src/main/js/components/dropdowns/jumplists.js

```diff
@@ -115,6 +55,29 @@ function generateDropdowns() {
       ),
   );
 
+  behaviorShim.specify(
+    ".dropdown-indicator",
+    "-clickable-dropdown-",
+    1000,
+    (element) =>
+      Utils.generateDropdown(
+        element,
+        createDropdownContent(
+          element,
+          element.getAttribute("data-model"),
+          element.getAttribute("data-children"),
+          element.getAttribute("data-href"),
+        ),
+        false,
+        {
+          trigger: "click focus",
```

Comment:
Why does this open the menu on focus rather than e.g on space bar?

Reply:
as explained earlier this is not how tippjs works. 

- https://github.com/jenkinsci/jenkins/pull/10803#issuecomment-3052719979
- https://github.com/jenkinsci/jenkins/pull/10803#issuecomment-3057777710

Reply:
seems like it could be triggered using `manual` from reading the second link?